packages/venia-concept/src/__tests__/AsyncWebpackPlugin.spec.js

```diff
@@ -0,0 +1,43 @@
+const AsyncWebpackPlugin = require('../AsyncWebpackPlugin');
+
+const tapPromise = jest.fn();
+
+const compiler = {
+    hooks: {
+        emit: {
+            tapPromise
+        }
+    }
+};
+
+beforeEach(() => {
+    process.env.MAGENTO_BACKEND_URL =
+        'https://venia-cicd-lrov2hi-mfwmkrjfqvbjk.us-4.magentosite.cloud/';
+    fetch.resetMocks();
```

Comment:
Can you add a comment in the code about what `fetch` is and where it comes from?

Who is actually mocking it?

Reply:
Got it. Done.